src/translib/transformer/xfmr_vrf.go

```diff
@@ -0,0 +1,517 @@
+package transformer
+
+import (
+        "errors"
+        log "github.com/golang/glog"
+        "github.com/openconfig/ygot/ygot"
+        "strings"
+        "translib/ocbinds"
+)
+
+type NwInstMapKey struct {
+        NwInstName    string
+        NwInstType    string
+}
+
+const (
+        MGMT_VRF_ENABLE        = "mgmtVrfEnabled"
+)
+
+const (
+        MGMT_VRF_NAME          = "mgmt-vrf-name"
+)
+
+var nwInstTypeMap = map[ocbinds.E_OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE] string {
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_UNSET: "",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE: "DEFAULT_INSTANCE",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2L3: "L2L3",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2P2P: "L2P2P",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2VSI: "L2VSI",
+        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF: "L3VRF",
+}
+
+/* Top level network instance table name based on key name and type */
+var NwInstTblNameMapWithNameAndType = map[NwInstMapKey]string {
+        {NwInstName: "mgmt", NwInstType: "L3VRF"}: "MGMT_VRF_CONFIG",
+        {NwInstName: "Vrf",  NwInstType: "L3VRF"}: "VRF",
```

Comment:
Pl. use VRF_TABLE here.

Reply:
same as above, the config db table name is "VRF"